ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -467,7 +472,7 @@ public void setNotifyOnInlineViewLayout(boolean notifyOnInlineViewLayout) {
   public void updateView() {
     @Nullable
     TextUtils.TruncateAt ellipsizeLocation =
-        mNumberOfLines == ViewDefaults.NUMBER_OF_LINES ? null : mEllipsizeLocation;
+        mNumberOfLines == ViewDefaults.NUMBER_OF_LINES || mAdjustsFontSizeToFit ? null : mEllipsizeLocation;
```

Comment:
I had to do this otherwise the ellipsis still appeared some times, it seemed to happen when there was whitespace at the end of the line. The only downside is the ellipsis won't show if we hit the minimum font scale, not sure how big of a deal this is.

Reply:
So to be clear, that means if there's a very long chunk of text and you hit the minimum font size, the text would be truncated without ellipses? That seems like a bug to me, though I probably wouldn't block this PR on it, but how does iOS handle it? How difficult do you think it is to fix both issues?

Reply:
Exactly, text would be truncated without ellipses. AFAIK it only happens on Android and seemed to happen when the text had room to render according to our measurements but the ellipsis would still show when drawing the text.

Since ellispsising should never happen when adjusting size to fit unless we hit the minimum size that seemed like an okay compromise since I'm pretty sure hitting minimum size is rare and the default is so small it would barely be noticeable.

I haven't investigated why it happens as it would probably require debugging deep inside android text rendering pipeline where the ellipsis rendering code is to figure out why it appears.